MonoGame.Framework/GameWindow.cs

```diff
@@ -146,13 +159,21 @@ protected void OnScreenDeviceNameChanged ()
 		}
 
 #if WINDOWS || WINDOWS_UAP || DESKTOPGL || ANGLE
-		protected void OnTextInput(object sender, TextInputEventArgs e)
-		{
-            EventHelpers.Raise(this, TextInput, e);
-		}
+		protected void OnTextInput(object sender, TextInputEventArgs e)
+		{
+            EventHelpers.Raise(this, TextInput, e);
+		}
+	    protected void OnKeyDown(object sender, InputKeyEventArgs e)
+	    {
+	        EventHelpers.Raise(this, KeyDown, e);
+	    }
+	    protected void OnKeyUp(object sender, InputKeyEventArgs e)

```

Comment:
Maybe just expose these as internal, so we don't need the additional helper methods in the GameWindow subclasses that just call this.